### ISSUE TITLE
[ENG][Quest[Hellfire Peninsula] Bloody Vengeance

### ISSUE THREAD

**Issue by Technowizard15** [Atlantiss/BugTracker#2100]
http://www.wowhead.com/quest=10250/bloody-vengeance

The ghosts don't disappear after each try and you have to do more damage then them to get the kill making it harder and harder to do enough damage before Urtrak is slain to get his kill counted toward the quest goal, especially for classes with low burst damage.

https://www.youtube.com/watch?v=WhDSZAoieDM

**Comment by Enay** [comment]
Confirming this, still an issue.

**Comment by aholler2** [comment]
Is this quest even possible?

**Comment by MercuricUma** [comment]
@aholler2 yes it's doable but you need to do more damage than those npcs. I did it some days ago with a hunter who needed it as well so I invited him and killed it. Luckily for us the npc spawned up the stairs and not next to the npcs
